pkg/common/loader.go

```diff
@@ -109,7 +109,11 @@ func (d *DefaultLoader) Load() (*kubekeyapiv1alpha2.Cluster, error) {
 		Registry: {hostname},
 	}
 	if d.KubernetesVersion != "" {
-		s := strings.Split(d.KubernetesVersion, "-")
+		ver := normalizedBuildVersion(d.KubernetesVersion)
+		if len(ver) < 2 {
```

Comment:
Use regex to check the version more better, like https://github.com/kubernetes/kubernetes/blob/89aaf7c02dd33415ac994cecfddca7c02ed75834/cmd/kubeadm/app/util/version.go#L153

Reply:
Thanks your advice. :)